docs/commerce/cart/extend_cart.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Extend cart with new options.
+edition: commerce
+---
+
+# Extend cart
```

Comment:
This doc should be part of the https://doc.ibexa.co/en/latest/commerce/cart/cart_api/ 😉